### PR TITLE
Added new Api calls for better pirwise support

### PR DESCRIPTION
**Pull request by Artemkaaas**
Signed-off-by: artem.ivanov <artem.ivanov@dsr-company.com>

### CODE REVIEW

File: wrappers/java/src/test/java/org/hyperladger/indy/sdk/pairwise/CreatePairwiseTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+public class CreatePairwiseTest extends IndyIntegrationTest {
+
+	private Wallet wallet;
+	private String walletName = "pairwiseWallet";
+	private String unknownDid = "NcYxiDXkpYi6ov5FcYDi1e";
+	private String myDid;
+	private String theirDid;
+
+	@Before
+	public void createWallet() throws Exception {
```
Comment (jovfer) [inline wrappers/java/src/test/java/org/hyperladger/indy/sdk/pairwise/CreatePairwiseTest.java]: There are a lot of same `@Before` and `@After` with creating simple wallet. So I suggest to create `class IndyIntegrationTestWithSingleWallet extends IndyIntegrationTest`

File: libindy/src/commands/pairwise.rs [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+        let pairwise_json = pairwise.to_json()
+            .map_err(|err|
+                CommonError::InvalidState(format!("Can't serialize Pairwise: {:?}", err)))?;
+
+        self.wallet_service.set(wallet_handle, &format!("pairwise::{}", their_did), &pairwise_json)?;
+        Ok(())
+    }
+}
+
```
Comment (jovfer) [inline libindy/src/commands/pairwise.rs]: @vimmerru Should be logic moved to `services::pairwise`?

File: libindy/src/services/signus/types.rs [hunk 3]
```diff
@@ -116,4 +116,23 @@ impl TheirDid {
 
 impl JsonEncodable for TheirDid {}
 
-impl<'a> JsonDecodable<'a> for TheirDid {}
\ No newline at end of file
+impl<'a> JsonDecodable<'a> for TheirDid {}
+
+#[derive(Serialize, Deserialize)]
+pub struct DidPair {
```
Comment (jovfer) [inline libindy/src/services/signus/types.rs]: @Artemkaaas I can`t find usage of it, please re-check.

**Comment by jovfer** [comment]
@vimmerru please review python

File: wrappers/java/src/test/java/org/hyperladger/indy/sdk/pairwise/CreatePairwiseTest.java (hunk 1 above)
Comment (vimmerru) [reply wrappers/java/src/test/java/org/hyperladger/indy/sdk/pairwise/CreatePairwiseTest.java]: Not sure that this PR is the right place for this refactoring. I suggest to merge this and add some technical debt task.

**Pull request merged**